Tests/iaas/entropy/entropy-check.py

```diff
@@ -358,7 +415,7 @@ def main(argv):
                             connect_kwargs={"key_filename": env.keyfile.name},
                         ) as fconn:
                             # need to retry because it takes time for sshd to come up
-                            retry(fconn.open, exc_type="NoValidConnectionsError")
+                            retry(fconn.open, exc_type="NoValidConnectionsError,TimeoutError")
```

Comment:
```suggestion
                            retry(fconn.open, exc_type="NoValidConnectionsError, TimeoutError")
```
Only thing I found (so basically nothing), rest LGTM.

Reply:
Oh right I overlooked the `"`, my fault.